src/shared-components/accordion/index.tsx

```diff
@@ -93,10 +104,13 @@ class Accordion extends React.Component {
     } = this.props;
 
     return (
-      <AccordionBox isOpen={isOpen} noBorder={noBorder} disabled={disabled}>
-        <TitleWrapper onClick={disabled ? noop : onClick} disabled={disabled}>
+      <AccordionBox isOpen={isOpen} noBorder={!!noBorder} disabled={!!disabled}>
+        <TitleWrapper
+          onClick={(event): void => !disabled && onClick(event)}
```

Comment:
```
onClick={(event): void => {
            if (!disabled) {
              onClick(event);
            }
          }}
```

If `disabled` is true then this will return false, which is incompatible with void. This was surfacing before because we were typing `onClick` as `Function`.

Reply:
done